### PR TITLE
001 anton navigation

### PR DESCRIPTION
**Pull request by gimly93**

### CODE REVIEW

File: .idea/codeStyleSettings.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          <option name="IF_BRACE_FORCE" value="3" />
+          <option name="DOWHILE_BRACE_FORCE" value="3" />
+          <option name="WHILE_BRACE_FORCE" value="3" />
+          <option name="FOR_BRACE_FORCE" value="3" />
+        </codeStyleSettings>
+      </value>
+    </option>
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
+  </component>
+</project>
```
Comment (Electr0n) [inline .idea/codeStyleSettings.xml]: we have eslint to check code style
no need this file

File: src/actions/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    };
+  } else {
+    return {
+      type: 'KIND_DEFAULT',
+      payload: kind.target.value
+
+    };
+  }
+
+}
```
Comment (Electr0n) [inline src/actions/index.js]: add endline

File: src/components/content/content.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+export default class Content extends Component {
+
+  render() {
+    return (
+      <Switch>
+        <Route exact path='/' component={Home}/>
+      </Switch>
+    );
+  }
+}
```
Comment (Electr0n) [inline src/components/content/content.js]: endline

File: src/components/nav/nav.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+      <header>
+        <nav>
+          <ul>
+            <li><Link to='/'>Главная</Link></li>
+          </ul>
+        </nav>
+      </header>
+    );
+  }
+}
```
Comment (Electr0n) [inline src/components/nav/nav.js]: endline

File: src/containers/places-filter/places-filter.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    );
+  }
+}
+
+PlacesFilter.propTypes = {
+  filterKind: PropTypes.func.isRequired,
+  kinds: PropTypes.array.isRequired
+};
+
+// export PlacesFilter;
```
Comment (Electr0n) [inline src/containers/places-filter/places-filter.js]: don't leave unnecessary comments

File: src/index.js [hunk 6]
```diff
@@ -1,8 +1,22 @@
...
+const createStoreWithMiddleware = applyMiddleware(ReduxPromise)(createStore);
 
 ReactDOM.render(
-  <App />,
+  <Provider store={createStoreWithMiddleware(reducers)}>
+    <BrowserRouter>
+      <App />
+    </BrowserRouter>
+  </Provider>
+  ,
```
Comment (Electr0n) [inline src/index.js]: do we need this comma?

File: src/reducers/reducer-filter-kind.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+      return defaultState.filter(state => {
+        return state.kind === action.payload;
+      });
+    case 'KIND_DEFAULT':
+      return defaultState;
+
+  }
+
+  return state;
+}
```
Comment (Electr0n) [inline src/reducers/reducer-filter-kind.js]: endline

File: src/reducers/reducer-kinds.js [hunk 8]
```diff
@@ -0,0 +1,8 @@
+export default function () {
+  return [
+    {title: ''},
+    {title: 'clinic'},
+    {title: 'pharmacy'},
+    {title: 'saloon'},
+  ];
+}
```
Comment (Electr0n) [inline src/reducers/reducer-kinds.js]: endline

Comment (Electr0n) [reply src/reducers/reducer-kinds.js]: also why components are containers now?

Comment (gimly93) [reply src/reducers/reducer-kinds.js]: Components that use redux called containers

**Pull request closed without merging**